lib/StackTransitioner.js

```diff
@@ -162,7 +162,7 @@ export default class StackTransitioner extends Component {
 
     if (
       this.isPanning ||
-      nextProps.animationType === NONE ||
+      routeAnimationType === NONE ||
```

Comment:
Should the `animationType` of the route be `undefined`, the rest of the code will still default to the Stack's chosen `animationType`, which in my use-case means "none"